### PR TITLE
Add support for ecmascript 7 and 8

### PR DESCRIPTION
**Pull request by MitMaro**

### CODE REVIEW

**Comment by MitMaro** [comment]
That pointed out a mistake in the config. the ecmascript-7 config is to enable parsing of ecmascript-7 JavaScript but without bringing in the rules that are attached to the ecmascript-8 config for async/await.

File: README.md [hunk 1]
```diff
@@ -59,6 +59,20 @@ An extension of `eslint-config-bluedrop` that adds support for eslint-babel.
 1. `npm install --save-dev eslint-plugin-babel`
 1. add `"extends": "bluedrop/babel"` to your ESLint config file
 
+### eslint-config-bluedrop/ecmascript-7
+
+An extension of `eslint-config-bluedrop` that adds support for ecmascript 7.
```
Comment (whymarrh) [inline README.md]: Nit: ECMAScript casing

File: README.md [hunk 2]
```diff
@@ -59,6 +59,20 @@ An extension of `eslint-config-bluedrop` that adds support for eslint-babel.
...
+
+1. Ensure you have installed the dependencies from General Setup
+1. add `"extends": "bluedrop/ecmascript-7"` to your ESLint config file
+
+### eslint-config-bluedrop/ecmascript-8
+
+An extension of `eslint-config-bluedrop` that adds support for ecmascript 8.
```
Comment (whymarrh) [inline README.md]: Nit: ECMAScript casing

File: README.md [hunk 3]
```diff
@@ -59,6 +59,20 @@ An extension of `eslint-config-bluedrop` that adds support for eslint-babel.
 1. `npm install --save-dev eslint-plugin-babel`
 1. add `"extends": "bluedrop/babel"` to your ESLint config file
 
+### eslint-config-bluedrop/ecmascript-7
+
+An extension of `eslint-config-bluedrop` that adds support for ecmascript 7.
+
+1. Ensure you have installed the dependencies from General Setup
+1. add `"extends": "bluedrop/ecmascript-7"` to your ESLint config file
```
Comment (whymarrh) [inline README.md]: Nit: capital 'A' to start the line

File: README.md [hunk 4]
```diff
@@ -59,6 +59,20 @@ An extension of `eslint-config-bluedrop` that adds support for eslint-babel.
...
+
+### eslint-config-bluedrop/ecmascript-8
+
+An extension of `eslint-config-bluedrop` that adds support for ecmascript 8. 
+
+1. Ensure you have installed the dependencies from General Setup
+1. add `"extends": "bluedrop/ecmascript-8"` to your ESLint config file
```
Comment (whymarrh) [inline README.md]: Nit: capital 'A' to start the line

**Comment by dvalentiate** [comment]
belated Thank you for this, I have used it!

**Pull request merged**